dashboard_documentation.md

```diff
@@ -69,6 +69,94 @@ Create New Site
    <img class="my-3" src="/images/plant_site.png"/>
 
 
+Create New Aggregate
+--------------------
+{: .anchor}
+
+Aggregates are created by first defining metadata for the aggregate, then
+associating existing observations with it. The aggregate metadata determines
+how aggregation will be performed and the characteristics of the resulting
+timeseries data.
+
+Observations are included in an aggregate between an *Effective From* and
+an *Effective Until* date defined by the user. Note that observations are
+expected to contain all values in their effective range. Any values missing
+from an observation during computation will cause a failure. To avoid this
+failure, users should submit `NaN`s where data is missing for their
+observations.
+
+1. Navigate to the aggregates listing page using the **Aggregates** link in
+   the left sidebar. At the top of the listing, click **Create new Aggregate**.
+
+   <img class="my-3" src="/images/aggregate_listing.png"/>
+
+2. Enter the metadata for your Aggregate. Observations added to the aggregate
+   must have the same variable as the aggregate and an interval length less
+   than or equal to that of the aggregate.
+
+   <img class="my-3" src="/images/aggregate_form.png"/>
+
+3. After submission, you will be redirected to an Aggregate page which displays
+   the observations that make up the aggregate (see 
+   [Add Observations to an Aggregate](#add-observations-to-an-aggregate)).
+
+   <img class="my-3" src="/images/aggregate_page_no_obs.png"/>
+
+
+### Add Observations to an Aggregate
+{: .anchor}
+
+Aggregates are defines by the observations it contains. Observations must be
```

Comment:
Change "Aggregates are defines by the observations it contains. Observations must be" to... Observations may be added to an Aggregate by following the steps below. Observations must already exist in the Arbiter before they...